hbase_sql/src/main/java/com/splicemachine/olap/OlapServerSubmitter.java

```diff
@@ -244,6 +244,24 @@ public void run() {
 
     }
 
+    private byte[] getKeepAlivePayload() {
+        DatabaseVersion version;
```

Comment:
Does _version_ need to be re-read on every call? Can it be an instance, or even static, variable similar to _OlapServerMaster_?

Reply:
I initialize a keepAlive message prototype once in the beginning with the version information and then use that to generate the payload with the time, so we only access the version once per OlapServer lifetime.